cmd/pebble/cmd_logs.go

```diff
@@ -107,7 +107,7 @@ func (cmd *cmdLogs) Execute(args []string) error {
 // Needed because signal.NotifyContext is Go 1.16+
 func notifyContext(parent context.Context, signals ...os.Signal) context.Context {
 	ctx, cancel := context.WithCancel(parent)
-	ch := make(chan os.Signal)
+	ch := make(chan os.Signal, 10)
```

Comment:
what is the wisdom behind this magic number? const/comment maybe?

Reply:
Fair call. The 10 was just a magic value >0 that I saw used in LXD code. Anything above 0 shuts up go vet, and other places in Pebble use 2 for the os.Signal channel, so I used that and added a clarifying comment.